<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_AuxIO_Screen</class>
 <widget class="QWidget" name="HNS_AuxIO_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LBL_Title">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>AUX I/O Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Aux1" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Aux2" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Aux3" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="HNS_Message_Preview" name="GV_Aux4" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>430</y>
     <width>230</width>
     <height>125</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INPUT 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INPUT 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INPUT 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INPUT 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Set_Msg1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Set_Msg2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Set_Msg3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Set_Msg4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>CHANGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Enable1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ENABLED</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Enable2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ENABLED</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Enable3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ENABLED</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Enable4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>510</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:1px;
	border-color:#707070;
	border-radius:12px;
}

QPushButton:checked{
	background-color:#1A3760;
	color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>ENABLED</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Back">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>710</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#FFFFFF;
	color:#414141;
	border-style:outset;
	border-width:3px;
	border-color:#1A3760;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Save">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>710</y>
     <width>163</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:#1A3760;
	color:#FFFFFF;
	border-radius:12px;
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Duration1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Duration2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>560</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Duration3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>600</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DURATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Duration4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>600</y>
     <width>132</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Message_Preview</class>
   <extends>QWidget</extends>
   <header>hns_message_preview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
